runtime/compiler/control/JITClientCompilationThread.cpp

```diff
@@ -1071,6 +1071,25 @@ handleServerMessage(JITServer::ClientStream *client, TR_J9VM *fe, JITServer::Mes
          client->write(response, arrayElementKnotIndex, knot->getPointerLocation(arrayElementKnotIndex));
          }
          break;
+      case MessageType::VM_isLambdaFormGeneratedMethod:
+         {
+         auto recv = client->getRecvData<TR_OpaqueMethodBlock *>();
+         client->write(response, fe->isLambdaFormGeneratedMethod(std::get<0>(recv)));
+         }
+         break;
+      case MessageType::VM_vTableOrITableIndexFromMemberName:
+         {
+         auto recv = client->getRecvData<TR::KnownObjectTable::Index>();
+         client->write(response, fe->vTableOrITableIndexFromMemberName(comp, std::get<0>(recv)));
+         }
+         break;
+      case MessageType::VM_getMemberNameFieldKnotIndexFromMethodHandleKnotIndex:
+         {
+         auto recv = client->getRecvData<TR::KnownObjectTable::Index, std::string>();
+         auto &memberNameStr = std::get<1>(recv);
+         client->write(response, fe->getMemberNameFieldKnotIndexFromMethodHandleKnotIndex(comp, std::get<0>(recv), &memberNameStr[0]));
```

Comment:
This message sends one parameter, but the receiver expects two parameters.

Reply:
Should be fixed now. This message is not triggered with current code, I think it should be sent once inliner changes from a different PR are applied